Makefile.include

```diff
@@ -20,6 +20,10 @@ RIOTPKG ?= $(RIOTBASE)/pkg
 RIOTPROJECT ?= $(shell git rev-parse --show-toplevel 2>/dev/null || pwd)
 RIOTPROJECT := $(abspath $(RIOTPROJECT))
 
+# add git tools directory to PATH so git can pick it up
+PATH:=$(PATH):$(RIOTBASE)/dist/tools/git
+export PATH
```

Comment:
The export is only valid for sub-processes if make.

-----Original Message-----
From: Oleg Hahm <notifications@github.com>
To: RIOT-OS/RIOT <RIOT@noreply.github.com>
Cc: Kaspar Schleiser <kaspar@schleiser.de>
Sent: So., 20 März 2016 14:12
Subject: Re: [RIOT] make: pkg: introduce git-cache (#5112)

> @@ -20,6 +20,10 @@ RIOTPKG ?= $(RIOTBASE)/pkg
>  RIOTPROJECT ?= $(shell git rev-parse --show-toplevel 2>/dev/null || pwd)
>  RIOTPROJECT := $(abspath $(RIOTPROJECT))
>  
> +# add git tools directory to PATH so git can pick it up
> +PATH:=$(PATH):$(RIOTBASE)/dist/tools/git
> +export PATH

I'm not sure I would like it if a build script would modify my PATH variable. Maybe you can store the old value and restore it after the build?

---
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub:
https://github.com/RIOT-OS/RIOT/pull/5112/files/8b363905f6854a7f841f884bf5b6e71c4010782a#r56764211